components/ExpandTestimony/ExpandTestimony.js

```diff
@@ -41,7 +41,10 @@ const ExpandTestimony = props => {
                   ? "Test"
                   : testimony.authorDisplayName) +
                 " - " +
-                testimony.publishedAt.toDate().toLocaleString() +
+                testimony.publishedAt
+                  .toDate()
+                  .toLocaleString()
```

Comment:
Use toLocaleDateString() instead of the substring

Reply:
done